test/realtime/data_status_alerter_test.exs

```diff
@@ -0,0 +1,45 @@
+defmodule Realtime.DataStatusAlerterTest do
+  use ExUnit.Case
+  import(ExUnit.CaptureLog)
```

Comment:
Did you mean to use the syntax with parens here? I didn't even realize that worked.

Reply:
That's funny, I think my editor put those parens in on its own initiative.